frontend/src/modules/entitydetails/components/Metadata.css

```diff
@@ -3,6 +3,7 @@
     color: #AAA;
     font-size: 0.9em;
     font-style: italic;
+    line-height: 22px;
```

Comment:
This adds extra `line-height` to all #metadata elements, but we should also add it to the source string.

You can fix this by fixing this:
https://github.com/mozilla/pontoon/pull/1051#discussion_r213642105

Reply:
I don't understand. The source string being in the #metadata component right now, this should work fine, right? When I move the source string outside of metadata, then I'll make sure it has the right line-height too.

Reply:
It doesn't work fine witht the current markup, because it increases the `line-height` of all `#metadata` elements, including the original string.

If you compare the style with prod, you'll notice that we should increase line-height only for the original string, but not for other `#metadata` elements.